src/data/ACTIONS/root/RPR.ts

```diff
@@ -280,6 +280,8 @@ export const RPR = ensureActions({
 
 	// -----
 	// Soul Gauge oGCDs
+	// Note:
+	//  all of these grant Reaver, but we only keep it on Gluttony since that's the only real use at level cap
```

Comment:
Not sure I'm fully understanding why that makes a difference, if it generates a status, it generates a status

Reply:
This is due to a bug with how timeline (and I guess prepull statuses) work when multiple actions define the status applied. Comment explains why it's specifically on Gluttony, beyond that this is in alignment with how other jobs handle a single status coming from multiple actions.